src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx

```diff
@@ -378,7 +378,8 @@ const StatusDescriptionAdd: React.FC = () => {
           ns: ["drawer"],
         })}
       </p>
-      <Input
+      <Textarea
+        minRows={1}
```

Comment:
Don't let `textarea` only have one line, it just seems `text input`.
I hope it can looks like this. You can give it two or three rows as default to let user know what they have written when they input a lot.
 
![截圖 2024-05-21 上午2 14 39](https://github.com/CAMPUS-NYCU/smart-campus/assets/32840701/469485d7-a876-44ea-bebf-0328e42460c3)

Reply:
OK, I had changed the minRows of add description field and edit description field. Also, I had modified the size of add photo button, in order to align with the text above. Bellow are how they look now.
![截圖 2024-05-27 晚上7 03 03](https://github.com/CAMPUS-NYCU/smart-campus/assets/107465871/d4744d9c-416f-43cf-bcba-b0e1e1df0ac1)
![截圖 2024-05-27 晚上7 03 16](https://github.com/CAMPUS-NYCU/smart-campus/assets/107465871/39d2cf1e-44b9-415d-a3f0-d32e8cb4dbaf)